### PR TITLE
use orchestrator idam secrets

### PR DESCRIPTION
**Pull request by mdayican**
Please remove this line and everything above and fill the following sections:

### Change description ###

add idam user-password secrets and client secret

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: bulk-scan/values.yaml [hunk 1]
```diff
@@ -61,7 +61,41 @@ bulk-scan-orchestrator:
...
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-probate-password
+      IDAM_USERS_SSCS_USERNAME:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-sscs-username
+      IDAM_USERS_SSCS_PASSWORD:
+        secretRef: bsp-orchestrator-creds
+        key: idam-users-sscs-password
     environment:
+      S2S_SECRET: AAAAAAAAAAAAAAAA
```
Comment (rkondratowicz) [inline bulk-scan/values.yaml:98]: ???

Comment (lgonczar) [reply bulk-scan/values.yaml:98]: Worth adding a comment explaining this value

Comment (mdayican) [reply bulk-scan/values.yaml:98]: idam.s2s-auth.secret
s2s runs is in the pod.
check the "rpe-service-auth-provider" config in the same file

Comment (mdayican) [reply bulk-scan/values.yaml:98]: it is used throughout the config

Comment (rkondratowicz) [reply bulk-scan/values.yaml:98]: is it needed for those users though?

Comment (lgonczar) [reply bulk-scan/values.yaml:98]: By adding a comment I meant a comment in code. Or maybe we could reference a global value here?

Comment (mdayican) [reply bulk-scan/values.yaml:98]: I think when we run the deployment these values are not using. after the flux deployment and test we will see . we can add all these to read.me after we run in demo and sure these values are correct.

Comment (mdayican) [reply bulk-scan/values.yaml:98]: > is it needed for those users though?

I think no , but chart is for all user.
they can suppress the values if they do not want to use . 
user can see the values in the product chart if we do not write them here they need to investigate microservice chart and find out the env vars.

**Review by rkondratowicz — APPROVED** [review]

**Pull request merged**